build/scripts/promote.sh

```diff
@@ -44,11 +44,21 @@ update_version_file() {
 }
```

Comment:
Seems that there are github actions that can create a Branch, commit, and stat a PR.

Reply:
can probably do that in my script to. Just don't have time to do it right now.